### PR TITLE
Adding A Linter

### PR DESCRIPTION
**Pull request by saivan**
This commit reformats the code so that it complies with the
standard linting style. Its currently a work in progress, but
it is meant to pave the way for linting in the build process

### CODE REVIEW

File: src/attr.js [hunk 1]
```diff
@@ -1,74 +1,69 @@
...
+  attr: function (a, v, n) {
     // act as full getter
     if (a == null) {
       // get an object of attributes
       a = {}
       v = this.node.attributes
-      for (n = v.length - 1; n >= 0; n--)
-        a[v[n].nodeName] = SVG.regex.isNumber.test(v[n].nodeValue) ? parseFloat(v[n].nodeValue) : v[n].nodeValue
-      
+      for (n = v.length - 1; n >= 0; n--) { a[v[n].nodeName] = SVG.regex.isNumber.test(v[n].nodeValue) ? parseFloat(v[n].nodeValue) : v[n].nodeValue }
```
Comment (Fuzzyma) [inline src/attr.js]: multiple lines again :)

File: src/element.js [hunk 2]
```diff
@@ -243,31 +243,31 @@ SVG.Element = SVG.invent({
...
       }
 
       // remove previously set data
       this.node.removeAttribute('svgjs:data')
 
-      if(Object.keys(this.dom).length)
-        this.node.setAttribute('svgjs:data', JSON.stringify(this.dom)) // see #428
+      if (Object.keys(this.dom).length) {
+        this.node.setAttribute('svgjs:data', JSON.stringify(this.dom))
+      } // see #428
```
Comment (Fuzzyma) [inline src/element.js]: comment should be one line above

File: src/line.js [hunk 3]
```diff
@@ -1,50 +1,52 @@
...
-, extend: {
+  extend: {
     // Get array
-    array: function() {
+    array: function () {
       return new SVG.PointArray([
-        [ this.attr('x1'), this.attr('y1') ]
-      , [ this.attr('x2'), this.attr('y2') ]
+        [ this.attr('x1'), this.attr('y1') ],
+       [ this.attr('x2'), this.attr('y2') ]
```
Comment (Fuzzyma) [inline src/line.js]: This looks odd. why is one line more indented`?

File: src/svg.js [hunk 4]
```diff
@@ -1,100 +1,99 @@
+/* eslint-disable */
```
Comment (Fuzzyma) [inline src/svg.js]: why disable linting here?

**Comment by saivan** [comment]
This closes #823

**Pull request merged**